expressEngine/db/db.js

```diff
@@ -1,14 +1,14 @@
-import { connect, disconnect } from "mongoose";
+import { connect, disconnect } from 'mongoose';
 
 const dbUrls = {
-  local   : "mongodb://localhost:27017/rnd",
+  local: 'mongodb://localhost:27017/rnd',
```

Comment:
@aman-successive Create a config file in utils folder and use that wherever needed

Reply:
i will do that sir